bitcoin-units/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "bitcoin-units"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>", "The Rust Bitcoin developers"]
+license = "CC0-1.0"
+homepage = "https://github.com/rust-bitcoin/rust-bitcoin/"
```

Comment:
`hompepage` shouldn't duplicate `repository` - just omit it. (I've been guilty of this too, learned not long ago.)

Reply:
Removed.